OpenMarket/OpenMarket/Extension/CALayer+.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  CALayer+.swift
+//  OpenMarket
+//
+//  Created by Eddy, marisol on 2022/05/18.
+//
+
+import UIKit
+
+extension CALayer {
+    func addSeparator() {
+        let separator = CALayer()
+        
+        separator.frame = CGRect.init(x: 10, y: frame.height - 0.5, width: frame.width, height: 0.5)
```

Comment:
```suggestion
        separator.frame = CGRect(x: 10, y: frame.height - 0.5, width: frame.width, height: 0.5)
```

주로 협업에서는 .init이라는 함수를 축약해서 사용하는 편입니다 :)

Reply:
축약으로 수정했습니다 :)

Reply:
감사합니다 ㅎㅎ 축약했습니다!